robot/EDA/resources/TriggerHandlersPageObject.py

```diff
@@ -0,0 +1,195 @@
+import time
+
+from selenium.webdriver.common.keys import Keys
+from cumulusci.robotframework.pageobjects import HomePage
+from cumulusci.robotframework.pageobjects import BasePage
+from cumulusci.robotframework.pageobjects import DetailPage
+
+from cumulusci.robotframework.pageobjects import pageobject
+from locators import trigger_handlers_locators
+
+
+
+@pageobject("Home", "TriggerHandlers")
+class TriggerHandlersHomePage(BasePage):
+    object_name = None
+
+    @property
+    def eda(self):
+        return self.builtin.get_library_instance('EDA')
+
+    def _is_current_page(self):
+        """ Verify we are on the Trigger Handlers page
+            by verifying that the header title is 'Trigger Handlers'
+        """
+        self.selenium.page_should_contain_element(
+            trigger_handlers_locators["header"],
+            message="Header with text 'Trigger Handlers' is not available on the page"
+        )
+
+    def check_if_element_exists(self, xpath):
+        """ Checks if the given xpath exists """
+        elements = int(self.selenium.get_matching_xpath_count(xpath))
+        return True if elements > 0 else False
+
+    def click_action_button(self):
+        """ Clicks on the 'Action' button in Contacts """
+        locator = trigger_handlers_locators["action"]
+        locator_close_action = trigger_handlers_locators["action_close"]
+
+        # In some cases, the action dialog opens by default,
+        # hence making sure it is closed, before opening it otherwise it would nullify the click command
+        if self.eda.check_if_element_exists(locator_close_action):
+            self.selenium.click_element(locator_close_action)
+
+        self.selenium.wait_until_page_contains_element(
+            locator,
+            error="'Action' button is not available on Contacts page"
+        )
+        self.selenium.click_element(locator)
+   
+    def open_app_launcher(self, app_name):
```

Comment:
The Salesforce library already has a keyword for launching an app from the app launcher. Why did you have to create your own keyword?

Reply:
Not using.  resolved.

Reply:
I removed this file.